homeworks/andrii.triasun_andrey-tryasun/homework_3/functions-arrays.js

```diff
@@ -0,0 +1,29 @@
+// 1.1 Square function
+const square = x => x * x;
+
+console.log(square(5));
+console.log(square(15));
+console.log(square(53));
+
+// 1.2 Function of creating an array from zero to the 'max' value with a 'step' between the elements
+const createArray = (max, step) => {
+    const array = [];
+    const length = max / step;
+    for (let i = 0; i <= length; i++) {
+        array[i] = i * step;
```

Comment:
use `push()` here

hint: for this task try to refactor to use `i += step` instead of `i++`